### ISSUE TITLE
Zork 1 Uncaught RuntimeError

### ISSUE THREAD

**Issue by ajmeese7** [caiiiycuk/js-dos.com#221]
Hey there, when I downloaded `zork1.jsdos` from the JS-DOS website and ran it on my local site I got the following error from `wdosbox.js` in the console:

```
Uncaught RuntimeError: null function or function signature mismatch
    at 0059b2d6:0x142de
    at 0059b2d6:0x10c1a
    at 0059b2d6:0x1106b2
    at 0059b2d6:0xff870
    at 0059b2d6:0x142de
    at 0059b2d6:0xfab0
    at 0059b2d6:0x73e7c
    at 0059b2d6:0x22400
    at 0059b2d6:0x11e29
    at 0059b2d6:0x11b431
```

The game freezes and nothing works, I experienced the same issue on the official DOS.zone site [here](https://dos.zone/zork-the-great-underground-empire-1984/). Do you have any suggestions for how to best move forwards?

**Comment by caiiiycuk** [comment]
Fixed in emulators@0.73.6

**Comment by caiiiycuk** [comment]
[js-dos v7.4.4](https://github.com/caiiiycuk/js-dos/releases/tag/v7.4.4)
